Makefile

```diff
@@ -0,0 +1,71 @@
+SHELL=/bin/bash
+DIR_NAME ?= $(notdir $(shell pwd))
+
+DOCKER_REGISTRY ?= registry.honestbee.com/
```

Comment:
I forgot that was in there :) I think the Makefile can be removed all together and just use an automated repository.